File: modulo1/segmento3/TrabajosPracticos/Apunte3/ejercicio2/java/Ejercicio2.java
import java.util.Scanner;

public class Ejercicio2 {
	public static void main(String args[]) {
		Scanner keyboard = new Scanner(System.in);

		System.out.print("Ingrese un caracter para identificar si es una vocal, consonante o un numero: ");
		String caract = keyboard.nextLine();
		
		switch (caract) {
			case "a":
			case "e":
			case "i":
			case "o":
			case "u":
				System.out.println("Ud. ingreso una vocal");
				break;

			case "b":
			case "c":
			case "d":
			case "f":
			case "g":
			case "h":
			case "j":
			case "k":
			case "l":
			case "m":
			case "n":
			case "ñ":
			case "p":
			case "q":
			case "r":
			case "s":
			case "t":
			case "v":
			case "w":
			case "x":
			case "y":
			case "z":
				System.out.println("Ud. ingreso una consonante");
				break;

			case "0":
			case "1":
			case "2":
			case "3":
			case "4":
			case "5":
			case "6":
			case "7":
			case "8":
			case "9":
				System.out.println("Ud. ingreso un numero");
				break;
			
			default:
				System.out.println("Ud. ingreso un simbolo");
				break;
		}
	}
}

File: modulo1/segmento3/Apunte4/ejercicio2/ejercicio2.py
caract = input("Ingrese un caracter para saber si es una vocal, consonante o un numero: ")

if (caract == '0' or
	caract == '1' or
	caract == '2' or
	caract == '3' or
	caract == '4' or
	caract == '5' or
	caract == '6' or
	caract == '7' or
	caract == '8' or
	caract == '9'
	):
	print("Ud. ingreso un numero")

elif (caract == 'a' or 
	  caract == 'e' or
	  caract == 'i' or
	  caract == 'o' or
	  caract == 'u'
	 ):
	 print("Ud. ingreso una vocal")

elif (caract == 'b' or
	  caract == 'c' or
	  caract == 'd' or
	  caract == 'f' or
	  caract == 'g' or
	  caract == 'h' or
	  caract == 'j' or
	  caract == 'k' or
	  caract == 'l' or
	  caract == 'm' or
	  caract == 'ñ' or
	  caract == 'p' or
	  caract == 'q' or
	  caract == 'r' or
	  caract == 's' or
	  caract == 't' or
	  caract == 'v' or
	  caract == 'w' or
	  caract == 'x' or
	  caract == 'y' or
	  caract == 'z' 
	 ):
	print("Ud. ingreso una consonante")

else: 
	print("Ud. ingreso un simbolo")
